src/Charts/chartBar.ts

```diff
@@ -150,4 +161,43 @@ export class ChartBar extends Chart {
       scale.paddingInner(options.paddingInner)
     }
   }
+
+  public onPointerOver() {
+    this.highlight
+      .select('rect')
+      .style('opacity', 1)
+      .style('fill', () => {
+        const element = this.group.select('rect')
+        if (element.node() === null) return
+        return window.getComputedStyle(element.node() as Element).getPropertyValue('fill')
+      })
+      .style('stroke', 'currentColor')
+  }
+
+  public onPointerOut() {
+    this.highlight.select('rect').style('opacity', 0)
+  }
+
+  public onPointerMove(x: number | Date, _xScale, yScale) {
```

Comment:
Is this highlight correct?:
![image](https://github.com/Deltares/fews-web-oc-charts/assets/144685504/0debebe0-4be6-46b5-9850-5c76eb3593b5)